mkdocs.yml

```
@@ -203,4 +203,18 @@ theme:
   logo: assets/logo.png
   name: material
   palette:
-    primary: teal
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: teal
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: teal
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
```

Fix spacing and remove comments to [pass the codegen action](https://github.com/argoproj/argo-cd/actions/runs/4274222709/jobs/7440682373).
```suggestion
  - media: "(prefers-color-scheme: light)"
    scheme: default
    primary: teal
    toggle:
      icon: material/brightness-7
      name: Switch to dark mode
  - media: "(prefers-color-scheme: dark)"
    scheme: slate
    primary: teal
    toggle:
      icon: material/brightness-4
      name: Switch to light mode
```